style.css

```diff
@@ -399,3 +394,102 @@ ul {
   line-height: 43px;
   text-align: center;
 }
+
+/* contact form */
+
+.contact-section {
+  display: flex;
+  width: 100%;
+  justify-content: center;
+  background: white;
+  padding: 24px;
+  flex-direction: column;
+}
+
+.contact-section h2 {
+  font-family: Roboto, sans-serif;
+  font-weight: 700;
+  color: #172b4d;
+  text-align: center;
+  font-size: 2em;
+  padding-top: 15vh;
+  margin-bottom: 10vh;
+}
+
+.contact-form {
+  margin-bottom: 10vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.contact-form input {
+  width: 100%;
+  border-bottom: 0.5px solid lightgray;
+  padding: 5vh 0 2vh 2vh;
+  margin-bottom: 2vh;
+}
+
+#error-message {
+  color: red;
+  display: block;
+}
+
+.contact-form textarea {
+  width: 100%;
+  height: 20vh;
+  background: #fbf8f7;
+  margin: 2vh 0;
+  padding: 2vh 2vh 0;
+}
```

Comment:
- [ ] Please adhere to the `Figma` design as closely as possible, I noticed that your placeholder or text font family is not consistent with the other input fields and makes sure the design looks unprofessional. Kindly take a look at the below table.

<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-05-03 18-12-48](https://user-images.githubusercontent.com/66476329/235919956-7a9f1a4b-bb8b-462c-830c-b474dc5182f4.png)

</td>
<td>

![Screenshot from 2023-05-03 18-12-34](https://user-images.githubusercontent.com/66476329/235920009-91d5057e-ac0f-4efd-88d5-7cf89e5ee73d.png)

</td>
</tr>
</table>